tests/parser/syntax/test_interfaces.py

```diff
@@ -0,0 +1,133 @@
+import pytest
+
+from vyper import (
+    compiler,
+)
+from vyper.exceptions import (
+    InvalidTypeException,
```

Comment:
@fubuloubu I think I have all the valid/failed interface syntax pegged in this test case now. Could you just give it a once over please :)